src/test-run/index.js

```diff
@@ -106,6 +106,31 @@ export default class TestRun extends Session {
         this.injectable.scripts.push('/testcafe-automation.js');
         this.injectable.scripts.push('/testcafe-driver.js');
         this.injectable.styles.push('/testcafe-ui-styles.css');
+
+        this.requestHooks = Array.from(this.test.requestHooks);
+
+        this._initRequestHooks();
```

Comment:
Do we need that function that contains only one line? I guess inline `this.requestHooks.forEach(hook => this._initRequestHook(hook));` call is good.

Reply:
It's debatable.
Compare two code parts. Which is more readable?
### 1
```js
        this.injectable.scripts.push('/testcafe-automation.js');
        this.injectable.scripts.push('/testcafe-driver.js');
        this.injectable.styles.push('/testcafe-ui-styles.css');

        this.requestHooks = Array.from(this.test.requestHooks);

        this._initRequestHooks();
```

### 2
```js
        this.injectable.scripts.push('/testcafe-automation.js');
        this.injectable.scripts.push('/testcafe-driver.js');
        this.injectable.styles.push('/testcafe-ui-styles.css');

        this.requestHooks = Array.from(this.test.requestHooks);

        this.requestHooks.forEach(hook => this._initRequestHook(hook));
```

Reply:
Personally, I Don't see much differences) 
In the first case you have to keep **in mind** that you need to call the `_initRequestHooks` function after the `this.requestHooks = ...`. In the second case it's obvious from the code.